src/divoom.js

```diff
@@ -63,3 +63,19 @@ module.exports.setDateTime = async date => {
     for (const buffer of d.messages.asBinaryBuffer())
         await write(buffer);
 };
+
+
+/**
+ * Display Image on the Divoom device
+ *
+ * @param {String} [filename] Filename to read the image from (absolute path)
```

Comment:
```suggestion
 * @param {String} filename Filename to read the image from (absolute path)
```
This argument shouldn't be optional?